WireDomain/Sources/WireDomain/Event Processing/ConversationEventProcessor/ConversationProtocolUpdateEventProcessor.swift

```diff
@@ -32,9 +32,10 @@ protocol ConversationProtocolUpdateEventProcessorProtocol {
 
 struct ConversationProtocolUpdateEventProcessor: ConversationProtocolUpdateEventProcessorProtocol {
 
-    func processEvent(_: ConversationProtocolUpdateEvent) async throws {
-        // TODO: [WPB-10175]
-        assertionFailure("not implemented yet")
+    let repository: any ConversationRepositoryProtocol
+
+    func processEvent(_ event: ConversationProtocolUpdateEvent) async throws {
+        try await repository.pullConversation(with: event.conversationID)
```

Comment:
Just a note: there's some logic inside the conversation local store that I think should be pulled up here (eventually), namely when the protocol changes we append a system message.

Reply:
yes, once we have the MessageRepository [PR](https://github.com/wireapp/wire-ios/pull/2096) merged, I will revisit the logic where we append a message to a conversation